### DISCUSSION TITLE
NPM New release

### DISCUSSION THREAD

**Discussion by juansan7** [exceljs/exceljs#2099] [Q&A]
I am having a problem with conditional formatting already fixed in master branch (https://github.com/exceljs/exceljs/pull/1815), I see that the changes are not include in the latest version of NPM (4.3.0), are there any plans to release a new version soon?

**Comment by ivanbiletskyi** [comment]
@guyonroche May we ask you to publish a new version of the library to npm? I am also waiting for this fix 🙏
